### PR TITLE
[jit][edge] Enable CALL instruction in lite interpreter.

### PR DESCRIPTION
**Pull request by zhxchen17**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #65972
* #65971
* #65970
* #65969
* #65968
* #65967
* #65966
* #65965
* __->__ #65964

Differential Revision: [D31326149](https://our.internmc.facebook.com/intern/diff/D31326149/)

Differential Revision: [D31326149](https://our.internmc.facebook.com/intern/diff/D31326149)

### CODE REVIEW

**Comment by iseeyuan** [comment]
Looks clean to me with frames and function table! Could you have an end-to-end test with export -> load -> run? 
At export stage, the inline should be disabled. 
I'm curious if there is any schema change in the bytecode format.

File: torch/csrc/jit/mobile/interpreter.h [hunk 1]
```diff
@@ -28,6 +29,7 @@ struct Code {
   std::vector<std::function<void(Stack&)>> operators_;
   std::vector<c10::IValue> constants_;
   std::vector<c10::TypePtr> types_;
+  std::vector<Function*> functions_;
```
Comment (cccclai) [inline torch/csrc/jit/mobile/interpreter.h:32]: Is it needed? Looks like they are not initialized/used

Comment (zhxchen17) [reply torch/csrc/jit/mobile/interpreter.h:32]: Good question! This is the place where we insert upgrader functions, so I suppose you initialize this :) But if that's not the plan I can update.

**Review by cccclai — APPROVED** [review]
LGTM. Thanks!

**Pull request closed without merging**